piccolo/query/methods/objects.py

```diff
@@ -173,6 +174,93 @@ def run_sync(self, *args, **kwargs) -> TableInstance:
         return run_sync(self.run(*args, **kwargs))
 
 
+class UpdateSelf:
+    """
+    This allows the user to update a single object - useful when the values
+    are derived from the database in some way.
+
+    For example, if we have the following table::
+
+        class Concert(Table):
+            name = Varchar(unique=True)
+            tickets_available = Integer()
+
+    And we fetch an object::
+
+        >>> concert = await Concert.objects().get(name="Amazing concert")
+
+    We could use the typical syntax for updating the object::
+
+        >>> concert.tickets_available += -1
+        >>> await concert.save()
```

Comment:
TODO - update this to use `Band.popularity` as the example column, as it already exists in the playground.